roles/initialisation/templates/ansible-hosts-multimaster.j2

```diff
@@ -159,6 +158,11 @@ openshift_cluster_monitoring_operator_alertmanager_storage_capacity="2Gi"
 # Set the v3.11+ console hostname
 openshift_console_hostname=console.{{ domainSuffix }}
 
+{% if ocpBranding is defined and ocpBranding and ocpBrandingUrl is defined %}
```

Comment:
Is there going to be a heat branch for this separately and a new (minor?) version released?

Reply:
The parameter has been added in openshift-heat in the keystone_v3 branch. We should do a release when they are merged (but I was hoping to include satellite tools 6.5 in that, so it might be a while.

Reply:
I've reverted the openshift-heat repo to satellite 6.3 so we can merge some branches. We are drowning in pull requests.